R/cookie-parser.R

```diff
@@ -4,7 +4,7 @@ cookieFilter <- function(req){
   forward()
 }
 
-#' @importFrom httpuv decodeURI
+#' @importFrom httpuv decodeURIComponent
```

Comment:
I like the `httpuv::` namespacing below.  Can you remove the `@importFrom httpuv` as they are not necessary now?